### PR TITLE
TST: add a real test for the APBBinFileHandler

### PR DESCRIPTION
**Pull request by mrakitin**
This PR adds an example dataset for testing of the `APBBinFileHandler` and the testing itself.

### CODE REVIEW

**Comment by mrakitin** [comment]
A note: once we rework the handler and the acquisition approach to have an array returned per a handler, the tests will be adjusted (i.e., no pandas will be used.)

File: pizza_box/tests/test_handlers.py [hunk 1]
```diff
@@ -1,2 +1,42 @@
...
+    return bin_file
+
+
+@pytest.fixture
+def txt_file():
+    txt_file = _get_file(basename, "txt")
+    return txt_file
+
+
+def test_handler_init(bin_file, txt_file):
```
Comment (jklynch) [inline pizza_box/tests/test_handlers.py:28]: Is `txt_file` going to be tested in this function?

**Review by jklynch — APPROVED** [review]

File: pizza_box/tests/test_handlers.py (hunk 1 above)
Comment (mrakitin) [reply pizza_box/tests/test_handlers.py:28]: Good point. I think it's not needed right now, as it's implicitly used inside the class (not an input yet, but probably we should change it).

Comment (mrakitin) [reply pizza_box/tests/test_handlers.py]: I removed them.

**Pull request merged**